src/App/layouts/MainLayout/components/Wrapper/Wrapper.tsx

```diff
@@ -0,0 +1,16 @@
+import classNames from 'classnames';
+import styles from './Wrapper.module.scss';
+
+export type WrapperProps = {
+  children: React.ReactNode;
+};
+
+const Wrapper: React.FC<WrapperProps> = (props) => {
+  const { children } = props;
+
+  const cn = classNames(styles['Wrapper']);
+
+  return <div className={cn}>{children}</div>;
+};
```

Comment:
можно не выносить в переменные, получится короче. и можно заюзать встроенный тип реакта `PropsWithChildren`
```suggestion
export type WrapperProps = {
  children: React.ReactNode;
};

const Wrapper: React.FC<React.PropsWithChildren> = ({ children }) => {
  return <div className={styles.Wrapper}>{children}</div>;
};
```

Reply:
Мне кажется, что с выносом просто читать легче, особенно, если дефолтные значения есть у пропсов.
Где проспсов мало, действительно, так лучше выглядит. Поменяю.